mm2src/coins/nft/storage/sql_storage.rs

```diff
@@ -528,6 +528,12 @@ fn get_transfers_with_empty_meta_builder<'a>(conn: &'a Connection, chain: &'a Ch
     Ok(sql_builder)
 }
 
+fn is_table_empty(conn: &Connection, table_name: &str) -> Result<bool, SqlError> {
+    let mut stmt = conn.prepare(&format!("SELECT COUNT(*) FROM {}", table_name))?;
+    let count: i64 = stmt.query_row([], |row| row.get(0))?;
+    Ok(count == 0)
```

Comment:
This should work without unwinding the result type

```suggestion
    stmt.query_row([], |row| row.get(0)).map(|t: i64| t == 0)
```

Reply:
removed `Ok(count == 0)`